### DISCUSSION TITLE
Volume Control feature

### DISCUSSION THREAD

**Discussion by tr3x33** [aldostools/webMAN-MOD#521] [Q&A]
Hello all,

Could you please let me know if there is a chance to control the volume level of a particular app/game or the whole PS3 from WebMAN? Maybe some plugin available?

At first sight it seems to be a simple thing, but as far as I understand there is still no implementation available.

Some games have volume level feature, how do they achieve this? 

How complicated would implement such a thing in Youtube App?

Thank you,
t

**Comment by aldostools** [comment]
I don't know how to do it, but as you say that it is a simple thing, post a pull request with the code to do it and I'll be glad to include it in webMAN MOD.
